server/models/Interview.mjs

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const interviewSchema = new mongoose.Schema({
+  interview_id: { type: Number, required: true, unique: true },
+  user_id: { type: Number, ref: 'User', required: true },
+  application_id: { type: Number, ref: 'Job_Application', required: true },
+  interview_date: { type: Date },
+  interview_time: { type: String },
```

Comment:
I believe the Date object will give you both the date and time, so we could combine this as interviewDateTime or interview_date_time (or whatever).

Reply:
Done